dev-client/src/translations/en.json

```diff
@@ -392,6 +392,7 @@
         "users": "users",
         "save": "Save",
         "save_fab": "Save",
+        "dismiss": "Dismiss",
```

Comment:
This doesn't seem to correspond to an entry in the es.json or es.po -- is that an issue?

Reply:
Yes, let me update es.json